xbmc/Application.cpp

```diff
@@ -2482,8 +2482,14 @@ void CApplication::FrameMove(bool processEvents, bool processGUI)
   {
     m_skipGuiRender = false;
     int fps = 0;
+
+    #if defined(TARGET_RASPBERRY_PI) || defined(HAS_IMXVPU)
```

Comment:
coding in a 80 cols terminal - you have to scroll on the very left to see what is going on ;-)